libs/community/langchain_community/chat_message_histories/postgres.py

```diff
@@ -9,28 +9,50 @@
     messages_from_dict,
 )
 
-logger = logging.getLogger(__name__)
+from psycopg2 import sql, pool, OperationalError
```

Comment:
Use `psycopg`  -- it's used for both version 1 and 3 I think (so it's newer than psycopg2)